spec/index.bs

```diff
@@ -1982,7 +1982,7 @@ dictionary GPUBindGroupLayoutEntry {
     boolean hasDynamicOffset = false;
 
     // Used for uniform buffer and storage buffer bindings.
-    GPUSize64 minBufferBindingSize = 0;
+    GPUSize64? minBufferBindingSize = null;
```

Comment:
nit: this could just be `GPUSize64 minBufferBindingSize`.

Reply:
```suggestion
    GPUSize64 minBufferBindingSize;
```

Reply:
Marking unresolved as this doesn't seem to have actually been done

Reply:
I committed this suggestion